gatling-http/src/main/scala/io/gatling/http/action/ws/Ws.scala

```diff
@@ -31,9 +31,11 @@ object Ws {
 
   def apply(requestName: Expression[String], wsName: Expression[String]): Ws = new Ws(requestName, wsName)
 
-  def checkTextMessage(name: String): WsFrameCheck.Text = WsFrameCheck.Text(name, Nil, Nil, isSilent = false)
+  def checkTextMessage(name: Expression[String]): WsFrameCheck.Text = WsFrameCheck.Text(name, Nil, Nil, isSilent = false, name = "")
+  def checkTextMessage(name: String): WsFrameCheck.Text = checkTextMessage(itemToElString(name))
```

Comment:
Please remove. You don't need to overload: a String parameter would implicitly be converted into an Expression[String]. That's how **all** the other methods in Gatling's DSL work.